packages/@sanity/cli/src/commands/codemod/codemodCommand.js

```diff
@@ -0,0 +1,116 @@
+/* eslint-disable no-process-exit, no-sync */
+const childProcess = require('child_process')
+const path = require('path')
+const fs = require('fs')
+const mods = require('../../actions/codemod/mods')
+
+const helpText = `
+Runs a given code modification script on the current studio folder.
+Running the command without a specified codemod name will list available transformations.
+
+Options
+  --dry Dry run (no changes are made to files)
```

Comment:
Really nice option, but I've seen many other CLIs call it `--dry-run`. Longer, but maybe consistent? No biggie.